lib/Steps/Integrations/Cordova.ts

```diff
@@ -190,31 +190,68 @@ export class Cordova extends BaseIntegration {
         shellPath: '/bin/sh',
         shellScript:
           // eslint-disable-next-line prefer-template
```

Comment:
Can we move this to a shell file that will ship with wizard and then copy the content to create the build phase?

For example `lib/Steps/Integrations/Cordova/SentryUploadDebugFiles.sh`?